bitnami/postgresql/templates/NOTES.txt

```diff
@@ -7,7 +7,7 @@ PostgreSQL can be accessed via port {{ template "postgresql.port" . }} on the fo
     {{ template "postgresql.fullname" . }}-read.{{ .Release.Namespace }}.svc.cluster.local - Read only connection
 {{- end }}
 
-{{- if and (not (eq .Values.postgresqlUsername "postgres")) (or .Values.postgresqlPostgresPassword (include "postgresql.useExistingSecret" .)) }}
+{{- if and (not (eq (include "postgresql.username" .) "postgres")) (or (include "postgresql.postgres.password" .) (include "postgresql.useExistingSecret" .)) }}
```

Comment:
Given the definition of the `postgresql.postgres.password` helper function:

https://github.com/bitnami/charts/blob/9084881e8330e01c49492243c29e144e77d4c530/bitnami/postgresql/templates/_helpers.tpl#L80-L91

Wouldn't it be `(include "postgresql.postgres.password" .)` always `true`?

Reply:
You are right.   `{{- if not (eq (include "postgresql.username" .) "postgres" }} `  will be more appropriate.

Reply:
I am not sure about that, it will be missing the condition to check that the password has been provided instead of auto-generated, that is the intention of the `(or .Values.postgresqlPostgresPassword (include "postgresql.useExistingSecret" .))` section

Reply:
I see. you tell me what is happen if we use values.yaml
`global:
  postgresql:
    postgresqlUsername: gg
    postgresqlPassword: gg
`
autogenerated secret - we have no problem, postgresPostgresPassword always exists. was fix #4123. 

Do you see any other error cases?

Reply:
it seems to be an old code that can be removed